index.js

```diff
@@ -155,49 +155,51 @@ function findSig(frame) {
   mod.wrap = (function (wrap) {
     return function (script) {
       script = mod.wrapper[0] + script + mod.wrapper[1];
-      
+
       var lines = script.split('\n');
 
-      var match = lines[line]
-        .substr(0, col)
-        .replace(comments, '')
-        .match(/function.+{/);
+      try {
```

Comment:
Wrapping this in a `try/catch` block puts the `match` variable inside this scope only. Underneath, which is outside of the `try/catch`, it references `match`, but `match` will always be `undefined` at that point.

Best to define `match` before the `try/catch`, so it can be accessed after the `try/catch`

```
let match = null;

try {
    match = ...
} catch {}

if (match) {
    ...
}
```